components/src/preact/map/sequences-by-location.tsx

```diff
@@ -13,24 +12,25 @@ import Info, { InfoComponentCode, InfoHeadline1, InfoParagraph } from '../compon
 import { LoadingDisplay } from '../components/loading-display';
 import { ResizeContainer } from '../components/resize-container';
 import { useQuery } from '../useQuery';
-import { type GeoJsonFeatureProperties, mapSourceSchema, useGeoJsonMap } from './useGeoJsonMap';
+import { mapSourceSchema } from './useGeoJsonMap';
 import { lapisFilterSchema, views } from '../../types';
 import Tabs from '../components/tabs';
 
-export const sequencesByLocationViewSchema = z.literal(views.map);
+export const sequencesByLocationViewSchema = z.union([z.literal(views.map), z.literal(views.table)]);
 export type SequencesByLocationMapView = z.infer<typeof sequencesByLocationViewSchema>;
 
 const sequencesByLocationPropsSchema = z.object({
     lapisFilter: lapisFilterSchema,
     lapisLocationField: z.string().min(1),
-    mapSource: mapSourceSchema,
+    mapSource: mapSourceSchema.optional(),
     enableMapNavigation: z.boolean(),
     width: z.string(),
     height: z.string(),
     views: z.array(sequencesByLocationViewSchema),
     zoom: z.number(),
     offsetX: z.number(),
     offsetY: z.number(),
+    pageSize: z.union([z.boolean(), z.number()]),
```

Comment:
```suggestion
    pageSize: z.union([z.boolean(), z.number()]).optional(),
```
Is this also an optional value, when users dont want to show the table?

The same for the zoom, offsetX, offsetY and enableMapNavigation.

Reply:
I think this is the first time i realized this, but the props of the view and the view list should be the same. Maybe we keep it like this and add a ticket to change this. I could imagine, that we put them into one prop:
```ts
...
lapisFilter: {dateFrom: .... },
lapisLocationField: "country",
views: [
  { type: "map",  mapSource: mapSourceSchema ...}
  { type: "table", pageSize: true }
]
...
```
But we would need to change this for all our other components as well. And we would need to have different props for the web component and the preact component. Or we also change this for the web component.

Reply:
As discussed, we'll leave it as it is.
* the props of the web component must match the Preact props so that the example code in the info box works
* We try to have primitive props when possible, because it's easier to use in HTML